pkg/daemon/update.go

```diff
@@ -1432,7 +1432,37 @@ func (dn *Daemon) enableUnits(units []string) error {
 	args := append([]string{"enable"}, units...)
 	stdouterr, err := exec.Command("systemctl", args...).CombinedOutput()
 	if err != nil {
```

Comment:
Shouldn't we do a check for RHEL 7?

Reply:
that's another option, I could hard check for RHEL7 and remove broken symlinks